webpack.config.js

```diff
@@ -221,6 +222,87 @@ const commonJsNoEncryptionConfig = {
   },
 };
 
+/**
+ * We create a bundle that exposes the mocha-junit-reporter package. We do this for the following reasons:
+ *
+ * - Browser:
+ *
+ *   1. This package is designed for Node only and hence requires polyfills of Node libraries (e.g. `stream`, `path`) — webpack takes cares of this for us.
+ *   2. The package is not compatible with RequireJS and hence we don’t have any easy way to directly load it in our tests — the webpack bundle exposes it as a global named MochaJUnitReporter.
+ *
+ * - Node: The library uses optional chaining syntax, which is not supported by Node 12.
+ */
+function createMochaJUnitReporterConfigs() {
+  const dir = path.join(__dirname, 'test', 'support', 'mocha_junit_reporter');
+
+  const baseConfig = {
+    mode: 'development',
+    entry: path.join(dir, 'index.js'),
+    module: {
+      rules: [
+        {
+          // The optional chaining syntax used by mocha-junit-reporter is not supported by:
```

Comment:
this part might be revisited when merging into v2 branch: we are using webpack 5 there, and it seems like we will be dropping support for node 12 and 14. maybe won't need `babel-loader` here at all

Reply:
Yep, I think we will be able to revisit this on that branch. Let's see where we are with the Node decision when we come to do the next merge into v2.